js/main.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-console */
+
+var $search = document.querySelector('#searching');
+$search.addEventListener('submit', getSearchText);
+
+function getSearchText(title) {
+  var searchText = title;
+  getShowId(searchText);
+}
```

Comment:
does this work? I would think that your submit event listener isn't actually working.